src/uiTest/resources/ideprobe-macos.conf

```diff
@@ -1,12 +1,11 @@
-# FIXME (#1383): revert to a single config file for macOS and Linux
+# FIXME (VirtusLab/ideprobe#332): revert to a single config file for macOS and Linux
 include "ideprobe-common.conf"
 
 probe {
-  intellij {
-    version {
-      # Note that DMGs are apparently published for EAPs and stable releases,
-      # but NOT for candidate EAPs (which our autoupdater in CI doesn't pick up anyway).
-      ext = ".dmg"
-    }
-  }
+  driver.vmOptions = ${probe.driver.vmOptions} [
```

Comment:
Values taken from `ps -f` for the IntelliJ executed by `./gradlew runIde`, then narrowed down to what's necessary.

Reply:
Works under 2023.3 and 2024.1